src/generateReadme.js

```diff
@@ -0,0 +1,113 @@
+const generateTitle = (answers) => {
```

Comment:
In each of these funtions, you can destructure your `answers` object, just like you have in the `getOtherContents` function. So here we would pull out `title` and `licence` since we are only using those keys. Please refactor the other functions too.

`const generateTitle = ( { title, licence } ) => { ....... etc }`